pkg/istio-agent/sds-agent.go

```diff
@@ -261,9 +268,13 @@ func (conf *SDSAgent) Start(isSidecar bool, podNamespace string) (*sds.Server, e
 	// we fail to load SDS
 	fail := conf.RequireCerts && conf.CertsPath == ""
 
+	var privateKey, certChain, rootCert []byte
+	if _, err := os.Stat(conf.JWTPath); os.IsNotExist(err) {
```

Comment:
why do we need this when we fail immediately below if we cannot read?

Reply:
This code is removed.